### DISCUSSION TITLE
LNK2019,无法解析的外部符号 FlagRegisterer

### DISCUSSION THREAD

**Discussion by jimesking** [PaddlePaddle/PaddleOCR#12359] [Q&A]
请提供下述完整信息以便快速定位问题/Please provide the following information to quickly locate the problem

- 系统环境/System Environment：windows10 x64, QT5.15.2
- 版本号/Version：Paddle：2.6  PaddleOCR：2.7.1 问题相关组件/Related components：args
- 运行指令/Command Code：build
- 完整报错/Complete Error Message：args.obj:-1: error: LNK2019: 无法解析的外部符号 "public: __cdecl google::FlagRegisterer::FlagRegisterer<bool>(char const *,char const *,char const *,bool *,bool *)" (??$?0_N@FlagRegisterer@google@@QEAA@PEBD00PEA_N1@Z)，函数 "void __cdecl fLB::`dynamic initializer for 'o_benchmark''(void)" (??__Eo_benchmark@fLB@@YAXXZ) 中引用了该符号

我们提供了AceIssueSolver来帮助你解答问题，你是否想要它来解答(请填写yes/no)?/We provide AceIssueSolver to solve issues, do you want it? (Please write yes/no): no

**Comment by Sunting78** [comment]
您好 可能需要在LINKER中添加依赖库。
